src/view/com/feeds/ProfileFeedgens.tsx

```diff
@@ -63,27 +63,27 @@ export const ProfileFeedgens = React.forwardRef<
     error,
     refetch,
   } = useProfileFeedgensQuery(did, opts)
-  const isEmpty = !isFetching && !data?.pages[0]?.feeds.length
   const {data: preferences} = usePreferencesQuery()
 
   const items = React.useMemo(() => {
+    const hasListData = data?.pages[0]?.feeds.length
     let items: any[] = []
-    if (isError && isEmpty) {
+    if (isError && !isFetching && !hasListData) {
       items = items.concat([ERROR_ITEM])
     }
     if (!isFetched && isFetching) {
       items = items.concat([LOADING])
-    } else if (isEmpty) {
+    } else if (isFetched && !hasListData) {
```

Comment:
Hiding the component when fetching is the cause of the bug, so stop making display dependent on `isFetching`

One side effect of this change is that the empty state will not hide during re-fetch from `ErrorMessage`.  Can address that if it seems like an issue